pkg/controller/release/release_controller.go

```diff
@@ -67,8 +72,7 @@ type Controller struct {
 	rolloutBlockLister shipperlisters.RolloutBlockLister
 	rolloutBlockSynced cache.InformerSynced
 
-	releaseWorkqueue     workqueue.RateLimitingInterface
-	applicationWorkqueue workqueue.RateLimitingInterface
```

Comment:
There's a comment a few lines above that says "Release Controller has 2 primary workqueues: releases and applications" - I guess that can go away now ^_^